### DISCUSSION TITLE
Extended feedback

### DISCUSSION THREAD

**Discussion by sofietoft** [GitbookIO/community#14] [Feature Requests]
### What's your feature/idea?

Extending the Feedback feature, with an option to add a note when rating an article.
Also, enable the option to subscribe to notifications for feedback notes being submitted.

Going even further, this could be tied into the Git Sync feature, by being able to "Submit an Issue" directly to the repo, like it's currently possible to "Edit on GitHub" when Git Sync is enabled! 🙌🏻

### What's your desired outcome?

I would like to be able to get more detailed feedback on the content we make.
We currently use the GitHub Issue tracker for this, but it would be great if our users could submit the feedback directly on the GitBook platform, instead of having to jump into GitHub and submit the issue there!

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by SamyPesse** [comment]
Generally, this is something we considered, and we'd rather stay away from doing it in the core product. While it could be easy to implement, it'd open the door to a lot of product features that we don't want to support for now: listing of these feedback notes, content moderation, ability to respond to the user, etc.

We might explore some related aspects in the coming months as we want to explore "External contributions", a feature to let visitors follow and contribute to the documentation.

> We currently use the GitHub Issue tracker for this, but it would be great if our users could submit the feedback directly on the GitBook platform, instead of having to jump into GitHub and submit the issue there!

This is probably the approach we'll explore to solve this specific problem of improving "feedbacks": integrations. Providing more custom blocks, from our GitHub integrations or others (Zendesk, etc.) to let visitors submit feedback to different platforms directly from your documentation.

**Reply by sofietoft** [reply]
Thanks for the detailed answer about this!

I completely understand the sentiment for not adding this to the core product. This is also not a vital feature for us, as we already have a feedback channel on the GitHub repo 🙌 

Using extensions/integrations is definitely also something we're open to. GitHub would be the obvious choice for us of course, but also Zendesk is interesting, as this is already a tool we use internally for our support channels 💪
